src/client/core.ts

```diff
@@ -32,8 +32,11 @@ export async function request<Req, Res>(options: ClientRequest<Req>, client: Cli
     "content-type": contentType ?? MimeType.JSON,
   };
 
-  if (overrides?.TOKEN) {
-    headers.Authorization = `Bearer ${overrides?.TOKEN}`;
+  // TODO - auth token is being used only for faucet, it breaks full node requests.
+  // Find a more sophisticated way than that but without the need to add the
+  // auth_token on every `aptos.fundAccount()` call
+  if (overrides?.AUTH_TOKEN && url.includes("faucet")) {
```

Comment:
We'll need to make a more complex auth token strategy for that then.

It's possible an node provider uses a token like this for authorization of requests